### DISCUSSION TITLE
Inquiry about Cell Isotropy and 'amrex::almostEqual()'

### DISCUSSION THREAD

**Discussion by yinmin-liu** [AMReX-Combustion/PeleLMeX#207] [Q&A] [answered]
Dear PeleLMeX Community,

I have a question regarding the **'amrex::almostEqual()'** function used in line 32 of the PeleLMSetup.cpp file. Could you please clarify how close the values of **'dx[0]'** and **'dx[1]'** need to be in order to pass the assert statement?

In my simulation, I am working with a 2D domain of size 0.008625 * 0.010065  and **'amr.n_cell = 576 672'**. However, when I check the cell sizes in the simulation output, they are reported as **'1.497395833e-05 * 1.497767875e-05'**.

![Screenshot 2023-05-17 at 12 54 09 PM](https://github.com/AMReX-Combustion/PeleLMeX/assets/55120383/639d3322-76ea-4e12-902f-f5a581bb1301)

I would appreciate any suggestions on how to make the cells more isotropic.

Thank you kindly!

**Comment by esclapez** [comment, marked as answer]
Hi there,

These must be equal to machine precision (so almost equal). So the only way to make the cell isotropic is to change your domain size (to keep the multiple of 32 in amr.n_cell). We could relax this constraint for cases that do not involve Embedded Boundary, but it is not very high on the development list at the moment.

**Reply by yinmin-liu** [reply]
Since my simulation doesn't include EB, would it be advisable to comment out line 32 in the PeleLMSetup.cpp for my implementation?
